draft-ietf-tls-tls13.md

```diff
@@ -2306,9 +2387,11 @@ values, each representing a single set of key agreement parameters;
 for instance a client might offer shares for several elliptic curves
 or multiple integer DH groups. The shares for each ClientKeyShareOffer
 MUST by generated independently. Clients MUST NOT offer multiple
-ClientKeyShareOffers for the same parameters. It is explicitly
-permitted to send an empty client_key_share extension as this is used
-to elicit the server's parameters if the client has no useful
+ClientKeyShareOffers for the same parameters. Servers receiving duplicate
+ClientKeyShareOffers SHOULD respond with an "illegal_parameter" alert
+and close the connection.
+It is explicitly permitted to send an empty "client_key_share" extension as
+this is used to elicit the server's parameters if the client has no useful
```

Comment:
I'll add one in here and for the other client-only extensions defined in this document that don't already have it.